phpcs-ruleset.xml

```diff
@@ -39,7 +39,7 @@
 
     <rule ref="Generic.Files.LineLength">
         <properties>
-            <property name="lineLimit" value="125"/>
+            <property name="lineLimit" value="135"/>
```

Comment:
```
public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false)
    {
FILE: /private/var/www/PHP-Parallel-Lint/src/Process/LintProcess.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 29 | ERROR | [x] The closing parenthesis of a multi-line function
    |       |     declaration must be on a new line
    |       |     (PEAR.Functions.FunctionDeclaration.CloseBracketLine)
 30 | ERROR | [x] The closing parenthesis and the opening brace of a
    |       |     multi-line function declaration must be on the same
    |       |     line
    |       |     (PEAR.Functions.FunctionDeclaration.NewlineBeforeOpenBrace)
```
    public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false
    )
    {
 31 | ERROR | [x] The closing parenthesis and the opening brace of a
    |       |     multi-line function declaration must be on the same
    |       |     line
    |       |     (PEAR.Functions.FunctionDeclaration.NewlineBeforeOpenBrace)
```
```
    public function __construct(
        PhpExecutable $phpExecutable,
        $fileToCheck,
        $aspTags = false,
        $shortTag = false,
        $deprecated = false
    ) {
Opening brace should be on a new line
    |       |     (Generic.Functions.OpeningFunctionBraceBsdAllman.BraceOnSameLine)
```
So, isn't possible with this ruleset